tools/remote-compare.sh

```
@@ -0,0 +1,99 @@
+fi
+
+echo '== Exact staging2 ↔ production checksum diff =='
+if diff -u "$STAGING_MANIFEST" "$PROD_MANIFEST"; then
+  echo 'STAGING_PROD_THEME_PARITY=PASS'
+else
+  echo 'STAGING_PROD_THEME_PARITY=FAIL'
+fi
+
+echo '== Deploy identity markers =='
+printf 'LOCAL_GIT_SHA='; git -C "$REPO_ROOT" rev-parse HEAD
+printf 'STAGING_DEPLOY_SHA='; ssh "$STAGING_SSH_ALIAS" "tr -d '\\r\\n' < '$STAGING_ROOT/$THEME_REL/.nvx-deploy-sha' 2>/dev/null || true"; echo
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)